conda/SPECS/conda.spec

```diff
@@ -78,32 +81,38 @@ sed -r -i 's/^(__version__ = ).*/\1"%{version}"/' conda/__init__.py
-pip3.8 install ruamel-yaml-%{ruamel_version}/ --no-deps --install-option="--prefix=%{buildroot}%{install_path}" --install-option="--install-scripts=%{buildroot}%{install_path}/bin"
-(cd tqdm-%{tqdm_version} && python3.8 setup.py install --prefix=%{buildroot}%{install_path} --install-scripts=%{buildroot}%{install_path}/bin)
+python3.8 -m pip install pluggy-%{pluggy_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install pycosat-%{pycosat_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install ruamel.yaml.clib-%{ruamelyamlclib_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install ruamel-yaml-%{ruamel_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install tqdm-%{tqdm_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install certifi-%{certifi_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}"
+python3.8 -m pip install urllib3-%{urllib3_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}"  
+python3.8 -m pip install idna-%{idna_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install charset-normalizer-%{charset_normalizer_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
+python3.8 -m pip install requests-%{requests_version}/ --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
 
 # Install conda
-python3.8 setup.py install --prefix=%{buildroot}%{install_path} --install-scripts=%{buildroot}%{install_path}/bin
+python3.8 -m pip install . --no-deps --no-build-isolation --prefix="%{buildroot}%{install_path}" 
 
 # Install conda_package_handling and six
 %{__mkdir_p} conda_package_handling
-tar xvf %{SOURCE6} -C conda_package_handling
+tar xvf %{SOURCE5} -C conda_package_handling
 %{__cp} -r conda_package_handling/lib/python%{python3_version}/site-packages/conda_package_handling %{buildroot}/%{install_path}/lib/python%{python3_version}/site-packages/
 
 # Set up source files
 # TODO: FIX csh FILE
 %{__mkdir_p} %{buildroot}/%{install_path}/etc/profile.d
 install -m 0644 -Dt %{buildroot}/%{install_path}/etc/profile.d/ conda/shell/etc/profile.d/conda.{sh,csh}
-sed -i '1s|^|export CONDA_EXE="%{install_path}/bin/conda"\nexport _CE_M=""\nexport _CE_CONDA=""\nexport CONDA_PYTHON_EXE="/usr/bin/python%{python3_version}"\n\n|' %{buildroot}/%{install_path}/etc/profile.d/conda.sh
+sed -i '1s|^|export CONDA_EXE="%{install_path}/bin/conda"\nexport _CE_M=""\nexport _CE_CONDA=""\nexport CONDA_PYTHON_EXE="/opt/ohpc/pub/apps/gnu9/python38/3.8.15/bin/python3.8"\n\n|' %{buildroot}/%{install_path}/etc/profile.d/conda.sh
```

Comment:
Perhaps we should use the python version variables here to make upgrading python more seamless?